fl4health/clients/scaffold_client.py

```diff
@@ -48,7 +48,6 @@ def __init__(
         self.client_control_variates_updates: Optional[NDArrays] = None  # delta_c_i in paper
         self.server_control_variates: Optional[NDArrays] = None  # c in paper
         self.optimizer: torch.optim.SGD  # Scaffold require vanilla SGD as optimizer
-        self.server_model_state: Optional[NDArrays] = None  # model state from server
```

Comment:
This wasn't used anywhere, so I killed it off

Reply:
Good call, dug around and also came to the conclusion this isn't being used anywhere.